content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -1,51 +1,107 @@
-can help you improve the quality of your services.
+Asking customers to rate their experience with :guilabel:`Helpdesk` support provides an opportunity
+to gauge team performance, and track customer satisfaction. Ratings can published to the portal,
+providing customers with a general overview of the team's performance.
 
-Set up
-======
+Enable customer ratings on a Helpdesk team
+==========================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable *Ratings on
-tickets*. The feature automatically adds a default email template on the non-folded *closing
-stage(s)* of that team.
+To enable :guilabel:`Customer Ratings` on a Helpdesk team, go to :menuselection:`Helpdesk -->
+Configuration --> Teams`. Select a team from the list and navigate to the settings page. Scroll to
+the :guilabel:`Performance` section, and check the box for :guilabel:`Customer Ratings`.
 
-.. image:: ratings/ratings_on_tickets.png
+.. image:: ratings/ratings-enable.png
    :align: center
    :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
          in Odoo Helpdesk
 
-To edit the email template and the stage(s) set as the closing ones, go to the Kanban view of your
-helpdesk team and click on *Settings*, then on *Edit Stage*.
+Set a ratings request email template on a stage
+===============================================
 
-.. image:: ratings/edit_stage.png
-   :align: center
-   :alt: Overview of a helpdesk team kanban view emphasizing the menu edit stage in Odoo Helpdesk
+To automatically request ratings from customers once their tickets have closed, an email template
+needs to be added to the appropriate stage.
+
+To choose a stage, go to :menuselection:`Configuration --> Stages`. Or, from a team's settings
+page, click :guilabel:`Set an Email Template on Stages` under :guilabel:`Customer Ratings`. Select
+one of the stages from the list to add a ratings request template.
+
+.. important::
+   Customers should only be asked to rate tickets once an issue has been resolved and their ticket
+   is closed. Therefore, a *ratings request* email should only be added to a stage that is
+   :guilabel:`Folded in Kanban`, as tickets in a *folded stage* are considered closed.
+
+On the stage's settings page, select `Helpdesk: Ticket Rating Request` in the :guilabel:`Email
+Template` field. This template has been pre-configured with ratings customers can use to provide
+feedback. To view the template, click the arrow button to the right of the field.
+
+Once the template has been added to the stage, it will automatically send a message when a ticket is
+moved to that stage. Customers will be asked to rate the support they received with colored icons.
 
-Now, once a ticket reaches the stage(s) designated as the *Closing Stage*, an email is sent to the
-customer.
+ - *Green smiling face* - Satisfied
```

Comment:
I would use the example block for these face descriptions https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#example